benchmark/c51/README.md

```diff
@@ -0,0 +1,35 @@
+# Categorical DQN (C51) Benchmark
+
+This repository contains instructions to reproduce our DQN experiments.
```

Comment:
Categorical DQN

Reply:
Yeah this README.md stuff is not maintainbale. I am going to fix this now, but in the future let's do it via #165